cloudbuild.yaml

```diff
@@ -249,8 +257,19 @@ steps:
         cd /workspace/applications/rag/tests
         python3 test_frontend.py "127.0.0.1:8081"
         echo "pass" > /workspace/rag_frontend_result.txt
+
+        cd /workspace/
+        sed -i 's/<username>/$$KAGGLE_USERNAME/g' $(find . -type f -name "rag-kaggle-ray-sql-latest.ipynb")
+        sed -i 's/<token>/$$KAGGLE_KEY/g' $(find . -type f -name "rag-kaggle-ray-sql-latest.ipynb")
+        gsutil cp $(find . -type f -name "rag-kaggle-ray-sql-latest.ipynb") gs://gke-aieco-rag-$SHORT_SHA-$_BUILD_ID/notebooks/
```

Comment:
Update to rag-kaggle-ray-sql-interactive.ipynb. Also no need to run `find`, it can just reference the absolute path of the notebook since we know it